apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -918,6 +934,10 @@ ul.sub-list {
     background: $white !important;
 }
 
+.d-none {
```

Comment:
class d-none is already predefined in bootstrap.css @prateek-giddh

Reply:
Done